packages/block-library/src/navigation-link/edit.js

```diff
@@ -219,21 +150,18 @@ function NavigationLinkEdit( {
 					{ isLinkOpen && (
 						<LinkControl
 							className="wp-block-navigation-link__inline-link-input"
-							onKeyDown={ handleLinkControlOnKeyDown }
-							onKeyPress={ ( event ) => event.stopPropagation() }
-							currentLink={ link }
-							onLinkChange={ updateLink( setAttributes, label ) }
-							onClose={ () => {
-								onCloseTimerId = setTimeout( () => setIsLinkOpen( false ), 100 );
-							} }
-							currentSettings={ [
-								{
-									id: 'opensInNewTab',
-									title: __( 'Open in new tab' ),
-									checked: opensInNewTab,
-								},
-							] }
-							onSettingsChange={ updateLinkSetting( setAttributes ) }
+							value={ link }
+							onChange={ ( {
+								title: newTitle = '',
+								url: newURL = '',
+								newTab,
+							} = {} ) => setAttributes( {
+								title: escape( newTitle ),
+								url: newURL,
+								label: label || escape( newTitle ),
```

Comment:
Would it make sense to escape the tile inside the LinkControl component?

Reply:
probably, but let's keep this for a separate PR as this one is growing.

Reply:
@youknowriad  Do you need me to create an Issue for this?

Reply:
Please yes :) thanks